modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -507,18 +509,29 @@ inline IE::Blob::Ptr extractRemoteBlob(IECallContext& ctx, std::size_t i) {
     cv::util::any any_blob_params = ctx.inFrame(i).blobParams();
     auto ie_core = cv::gimpl::ie::wrap::getCore();
 
-    using ParamType = std::pair<InferenceEngine::TensorDesc,
-                                InferenceEngine::ParamMap>;
+    using ParamType = std::pair<std::pair<InferenceEngine::TensorDesc,
```

Comment:
Alignment is broken. Also, I recommend to add another alias, e.g.  `using ParamType = std::pair<InferenceEngine::TensorDesc, InferenceEngine::ParamMap>` and then `using NV12ParamType = std::pair<ParamType, ParamType>`

Reply:
And what about support Yuv420 for example or other types? Looks like we shall redesign it further in case of extending.
Also is it expected that legacy behavior (non NV12) is broken?

In order to keep it backward compatible and future extendable i would recommend to use something like `std::vector<pair>` instead or custom structure

Reply:
Well, there is no legacy here. Currently it's being used in a single case which slightly changed it's behavior, so we are replacing the default blob with NV12 one. I don't think we need to preserve backward compatibility in this PR, better to redesign it here to align with possible future cases

Reply:
ok then. I'll try to modify my own PR with expected implementation, but let's try on idea about extensibility

Reply:
Done